doc/cephfs/cephfs-top.rst

```diff
@@ -63,6 +63,60 @@ By default, `cephfs-top` uses `client.fstop` user to connect to a Ceph cluster::
   $ ceph auth get-or-create client.fstop mon 'allow r' mds 'allow r' osd 'allow r' mgr 'allow r'
   $ cephfs-top
 
+Description of Fields
+---------------------
+
+chit : cap hit
+Percentage of caps issued over total number of caps
+
+dlease : dentry lease
+Percentage of dentry leases handed out over the total dentry lease requests
+
+ofiles : opened files
+Number of opened files
+
+oicaps : pinned caps
+Number of pinned caps
+
+oinodes : opened inodes
+Number of opened inodes
+
+rtio : total size of read IOs
+Number of bytes read in input/output operations generated by all process
+
+wtio : total size of write IOs
+Number of bytes written in input/output operations generated by all processes
+
+raio : average size of read IOs
+Mean of number of bytes read in input/output operations generated by all process over total IO done
+
+waio: average size of write IOs
+Mean of number of bytes written in input/output operations generated by all process over total IO done
+
+rsp : read speed
+Speed of read IOs compared with the last refresh
+
+wsp : write speed 
+Speed of write IOs compared with the last refresh
+
+rlatavg : average read latency
+Mean value of the read latencies
+
+rlatsd : standard deviation (variance) for read latency
+Dispersion of the metric for the read latency relative to its mean
```

Comment:
![image](https://github.com/ceph/ceph/assets/23738514/4f1dbfc9-877c-426d-92d7-d38da6f2166e)

Can't differentiate the `abbreviation`, `expansion phrase` and the `description`. Could you please bring a visual difference between these (for all fields)?

Reply:
PTAL

Reply:
Do the same for description too. Something like https://ceph--55465.org.readthedocs.build/en/55465/cephfs/cephfs-top/#interactive-commands would be readable.

Reply:
I did the same way it is done for `interactive commands`, pls have  a look.

Reply:
Please see the build https://ceph--55465.org.readthedocs.build/en/55465/cephfs/cephfs-top/#description-of-fields